### PR TITLE
Fix version hardcoded in tarball generation test

### LINKED ISSUE #134: Tarball generation test has version hardcoded
**Describe the bug**

Currently, the tarball generation tests have the [version harcoded.](https://github.com/unitycatalog/unitycatalog/blob/main/tests/test_tarball_generation.py#L13) This would break as we make releases.

**To Reproduce**

1. Change version in build.sbt
2. Run tests/test_tarball_generation.py

**Expected behavior**

The version to be used should be picked up from build.sbt (or at least should not be hardcoded)

**System [please complete the following information]:**
-   OS: MacOS Sonoma 14.4.1

**Additional context**

### PR DESCRIPTION
**Pull request by tlm365**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [x] If there is a related issue, make sure it is linked to this PR.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
Closes #134.

### CODE REVIEW

**Review by vikrantpuppala — APPROVED** [review]
That was quick! Thanks for fixing this @tlm365, @tdas, could we please merge?

File: tests/test_tarball_generation.py [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+import signal
+import subprocess
 import sys
+import tarfile
+import time
+
 
 # 1. Extract the tarball
-tarball_path = "target/unitycatalog-0.1.0.tar.gz"
+tarball_path = "target/unitycatalog-{version}.tar.gz"
```
Comment (tdas) [inline tests/test_tarball_generation.py:15]: will this work with version being `x.y.z-snapshot`?

Comment (tdas) [reply tests/test_tarball_generation.py:15]: @tlm365

Comment (tdas) [reply tests/test_tarball_generation.py:15]: I think it should.

**Review by tdas — APPROVED** [review]
thank you for the quick fix @tlm365 . merging this.

File: tests/test_tarball_generation.py (hunk 1 above)
Comment (tlm365) [reply tests/test_tarball_generation.py:15]: @tdas as long as it's in quotes (in `version.sbt`) -> version := "**this_will_be_captured**"

**Pull request merged**